src/provvis.ts

```diff
@@ -358,13 +358,13 @@ class StateRepr {
     $body.select('textarea').on('input', function() {
       $body.select('input.readonly').property('value', extractTags(this.value).join(' '));
     });
-
-    dia.onHide(() => {
+    $(dia.footer).find('button.btn-primary').on('click', function () {
```

Comment:
```suggestion
    dia.footer.querySelector('button.btn-primary').addEventListener('click', function () {
```
I suggest to use the native JS API instead of using jQuery. Please test if this works as well.